v2_1/ws/rest/docs/4_7_errors.md

```diff
@@ -5,11 +5,11 @@ RESTful web services use HTTP status codes to indicate errors. In principle, any
 HTTP status code | Description
 ---|---
 200 | This is the standard response for successful HTTP requests. With `GET` requests, it is used to indicate that the request was successfully processed and that the data are available in the body of the response.
+204 | If the result from the query is empty the web service should return this message. `204` is preferred to `404` in this case, as a lack of results matching a query is an “empty but still healthy state”.
```

Comment:
Should it read like this?
`If the result from the query is empty, the web service should return **this code**`
In the sense that the response is not a message, but only an HTTP code.